### PR TITLE
docs: update readme

### LINKED ISSUE #33: Readme has broken links and is missing usage instructions
## Describe the bug

The README.md file has a number of links that go nowhere, and it is also missing instructions for how to use ACPaaS UI.

## Expected behavior

All links go somewhere sane, clear instructions are given on how to get started using ACPaaS UI.

### PR DESCRIPTION
**Pull request by jsebrech**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #33

## What is the new behavior?

- Fixed broken links
- Added usage instructions
- Added descriptions for the components
- Wording changes

*Known issue:* the slack url points to the closed slack space. This will be addressed as part of #28.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Resolved issues

Fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,74 +16,84 @@ This library is part of [ACPaaS UI][acpaas-ui].
...
+ng new my-app --style=scss
+cd my-app
+npm install @a-ui/core @acpaas-ui/ngx-components
+```
+
+Edit styles.scss to import the branding look & feel.
+
+```scss
+/* core branding, https://a-ui.github.io/core_branding_scss/ */
+@import '@a-ui/core/dist/main.css';
```
Comment (jsprds) [inline README.md]: Import core_branding only for variables / functions / mixins / ... the main styling should be loaded through the CDN as described on: https://github.com/a-ui/core_branding_scss unless there's a specific project-related reason not to do it.

Comment (TriangleJuice) [reply README.md]: I would also prefer "the CDN way" to be documented here and maybe link to the core branding for more advanced setups.

File: README.md [hunk 2]
```diff
@@ -18,74 +16,84 @@ This library is part of [ACPaaS UI][acpaas-ui].
...
+| localstorage   | Interface with localstorage              | [Documentation](./packages/localstorage/README.md)        |
+| logo           | Logo icon                                | [Documentation](./packages/logo/README.md)                |
+| map            | Leaflet-based map                        | [Documentation](./packages/map/README.md)                 |
+| notifications  | Show user notifications                  | [Documentation](./packages/notifications/README.md)       |
+| pagination     | Pagination control                       | [Documentation](./packages/pagination/README.md)          |
+| progress-bar   | Progress bar control                     | [Documentation](./packages/progress-bar/README.md)        |
+| selectable-list| List control with selectable items       | [Documentation](./packages/selectable-list/README.md)     |
+| table          | Interactive table                        | [Documentation](./packages/table/README.md)               |
+| utils          | Assorted helpers                         | [Documentation](./packages/utils/README.md)               |
```
Comment (TriangleJuice) [inline README.md]: Maybe we should also list the submodules, like auto-complete?

Comment (jsebrech) [reply README.md]: Should be added here when documenting the forms module (links to titles inside the forms readme).

File: README.md (hunk 1 above)
Comment (jsebrech) [reply README.md]: changed it to the CDN way

File: README.md (hunk 2 above)
Comment (TriangleJuice) [reply README.md]: True. Good call.

**Pull request merged**